iot_config/tests/route_service.rs

```diff
@@ -195,7 +195,7 @@ async fn stream_only_sends_data_modified_since(pool: Pool<Postgres>) {
     let response = client
         .stream(route_stream_req_v1(
             &client_keypair,
-            since.timestamp_millis() as u64,
+            since.timestamp() as u64,
```

Comment:
the description says responses will now be in millisecs, but yet this and others are changed to secs ?